AzureDevOps.Authentication/Src/MsaAuthentication.cs

```diff
@@ -33,8 +33,7 @@ namespace AzureDevOps.Authentication
 
     public sealed class MsaAuthentication : Authentication, IMsaAuthentication
     {
-        public const string DefaultAuthorityHost = AzureDevOps.Authentication.Authority.AuthorityHostUrlBase + "/live.com";
-        internal const string QueryParameters = "domain_hint=live.com&display=popup&site_id=501454&nux=1";
```

Comment:
We can avoid domain_hint along with the change to move to "common". But not sure about other parameters, hopefully you've verified that those are also not required here.

Reply:
Thanks @sksk18 things seem to work without them.  The changes were added in this [commit](https://github.com/Microsoft/Git-Credential-Manager-for-Windows/commit/612389ec9708fa7343e99fdb2947c4e91046afc5) which doesn't give much context.

It's probably safer and easier to just keep things scoped to my change so I'll update to simply drop domain_hint.  Thanks for the tip!  @mjcheetham let me know if you have any thoughts here.